src/backend/PortabilityService.Gateway/Program.cs

```diff
@@ -52,7 +54,7 @@ private static void ConfigureLogging(WebHostBuilderContext context, ILoggingBuil
 
             // Create Serilog configuration from app configuration
             var serilogLoggerConfiguration = new LoggerConfiguration()
-                .ReadFrom.Configuration(config);
+                .ReadFrom.Configuration(config.GetSection(ConfigurationProviderConstants.PortabilityServiceConfigurationRoot));
```

Comment:
It would be better to just read from the configuration directly (rather than a config service-specific section) so that if we end up falling back to appsettings.json (if the config service is down or something) everything just works.

Reply:
Updated the ConfigService to return the section settings without nulls and settings. Now this will just be Config instead of Config.GetSection